src/app/home-page/widget/organization-box/organization-box.component.ts

```diff
@@ -38,45 +39,22 @@ export class OrganizationBoxComponent extends Base implements OnInit {
 
   ngOnInit(): void {
     this.getMyOrganizations();
-    this.requestsQuery.myPendingOrganizationRequests$.subscribe(
-      (requests) => {
-        if (requests) {
-          this.pendingRequests = requests;
-        }
-      },
-      (error: HttpErrorResponse) => {
-        this.alertService.detailedError(error);
-      }
-    );
-    this.requestsQuery.myOrganizationInvites$.subscribe(
-      (invites) => {
-        if (invites) {
-          this.pendingRequests = invites;
-        }
-      },
-      (error: HttpErrorResponse) => {
-        this.alertService.detailedError(error);
-      }
-    );
+    this.pendingRequests$ = this.requestsQuery.myPendingOrganizationRequests$;
+    this.pendingInvites$ = this.requestsQuery.myOrganizationInvites$;
   }
 
   private getMyOrganizations() {
     this.usersService
-      .getUserDockstoreOrganizations()
+      .getUserDockstoreOrganizations(null, this.filterText)
       .pipe(
         finalize(() => (this.isLoading = false)),
         takeUntil(this.ngUnsubscribe)
       )
       .subscribe(
         (myOrgs: Array<OrganizationUpdateTime>) => {
-          this.listOfOrganizations = [];
-          myOrgs.forEach((org: OrganizationUpdateTime) => {
-            if (this.listOfOrganizations.length < 5) {
-              this.listOfOrganizations.push(org);
-            }
-            if (this.firstCall) {
-              this.totalOrgs += 1;
-            }
+          myOrgs.forEach(() => {
+            this.listOfOrganizations = myOrgs.slice(0, 7);
+            this.totalOrgs = myOrgs.length;
```

Comment:
This is going to be the total of filtered organizations you belong to, not the total organizations you belong to. So if belong to at least one org, but type in a filter that doesn't match any of your orgs, you're going to get the message `You are not in any organizations`, which seems wrong.